actionview/lib/action_view/helpers/tags/base.rb

```diff
@@ -114,7 +114,7 @@ def sanitized_method_name
         end
 
         def sanitized_value(value)
-          value.to_s.gsub(/\s/, "_").gsub(/[^-\w]/, "").downcase
+          value.to_s.gsub(/\s/, "_").gsub(/[^-\p{Word}]/, "").downcase
```

Comment:
This is now implemented on master. You can remove this change and its associated test.